opensaber-client/src/main/java/io/opensaber/registry/client/Client.java

```diff
@@ -74,4 +74,16 @@
      * @return
      */
     ResponseData<T> deleteEntity(URI property, Map<String, String> headers) throws ClientProtocolException, IOException, URISyntaxException ;
+    
+    /**
+     * This method will allow you to search for entities based on property names and values provided in the JSON input
```

Comment:
This method documentation needs more info - please add what is the expected format of a RequestData? Alternatively, you can add a testcase too. Maybe I'd like to see if these exceptions are really valid (sometimes newer code would not throw these defined types, but generic Exception - would like to check on that).

Reply:
:+1:

Reply:
:+1: . Updated the API comments

Reply:
I thought we discussed to write like "\"teacher\":  { \"serialNum\": 12}" for easy understanding. Not a problem. Approving.